### PR TITLE
Sentiment visualization with Wordclouds

### PR DESCRIPTION
**Pull request by samrat-halder**
by Samrat Halder and Hariz Johnson

### CODE REVIEW

File: sentiment_with_wordcloud.Rmd [hunk 1]
```diff
@@ -0,0 +1,89 @@
+---
```
Comment (bl2788) [inline sentiment_with_wordcloud.Rmd]: should not contain a YAML header (the section at the top of the .Rmd that includes name, title, date, output, etc.) including the --- line.

File: sentiment_with_wordcloud.Rmd [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+title : Visualizing Movie Reviews in Word Cloud
+output: html_document
+author: Samrat Halder and Hariz Johnson
+---
+
+```{r , include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+#
```
Comment (bl2788) [inline sentiment_with_wordcloud.Rmd]: The first line starts with a single hashtag (#) followed by white space and a project name.

Comment (bl2788) [reply sentiment_with_wordcloud.Rmd]: Second line should be blank.

**Review by bl2788 — CHANGES_REQUESTED** [review]
1.You should not contain a YAML header at the start of your file.
2.The first line should starts with a single hashtag followed by white space and a project name.
3.The second line should be blank.

File: sentiment_with_wordcloud.Rmd (hunk 2 above)
Comment (samrat-halder) [reply sentiment_with_wordcloud.Rmd]: Resolved it. Please check.

File: sentiment_with_wordcloud.Rmd (hunk 1 above)
Comment (samrat-halder) [reply sentiment_with_wordcloud.Rmd]: I cannot see this line "@@ -0,0 +1,89 @@" in my .rmd file . Can you please check again?

Comment (samrat-halder) [reply sentiment_with_wordcloud.Rmd]: Removed the YAML header @bl2788  Please check

Comment (bl2788) [reply sentiment_with_wordcloud.Rmd]: You still have that YAML header.
I did not see any changes.

File: sentiment_with_wordcloud.Rmd (hunk 2 above)
Comment (bl2788) [reply sentiment_with_wordcloud.Rmd]: The first line means that the first line in the whole file. After you remove all the YAML header. The first line in your file should be a single hashtag followed by white space and a project name. The second line should be blank.

**Review by bl2788 — CHANGES_REQUESTED** [review]
I cannot see any changes. Please see the comments in the file.

File: sentiment_with_wordcloud.Rmd (hunk 1 above)
Comment (samrat-halder) [reply sentiment_with_wordcloud.Rmd]: Sorry, I forgot to push the commit. Can you check now?

File: sentiment_with_wordcloud.Rmd [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+removeURL <- function(x) {
+  gsub("http[[:alnum:]]*", "", x)
+}
+removelongWORDS <- function(x) {
+  gsub("\\b[[:alpha:]]{15,}\\b", "", x, perl=FALSE)
+}
+removeCharacters <-function (x, characters)  {
+  gsub(sprintf("(*UCP)(%s)", paste(characters, collapse = "|")), "", x, perl = FALSE)
+}
+reviews <- read.csv("reviews.csv", row.names = NULL)
```
Comment (skyetim) [inline sentiment_with_wordcloud.Rmd]: You should upload the dataset as well.

**Review by skyetim — CHANGES_REQUESTED** [review]

File: sentiment_with_wordcloud.Rmd (hunk 3 above)
Comment (samrat-halder) [reply sentiment_with_wordcloud.Rmd]: Added the files to resources. Please check and let me know if now it is fine.

**Comment by bojum** [comment]
🎉 Merged. Thanks for your contribution!

**Pull request merged**